proposals/lip-0061.md

```diff
@@ -133,6 +133,7 @@ In this LIP, we will frequently use the following functions specified in LIP 005
 - [`existBFTParameters`][lip-0058#existbftparameters]
 - [`getBFTHeights`][lip-0058#getbftheights]
 - [`getBFTParameters`][lip-0058#getbftparameters]
```

Comment:
I would remove this function and not use it any more (there is only one use which can also be replaced by `getBFTParametersActiveValidators`).

Reply:
Done. Indeed it can be removed, I just left the usage of `getBFTParameters` in `selectAggregateCommit`, since we agreed that it was optional to change it and it might be easier to catch future errors in debugging.

But indeed, makes sense to completely skip it and assume in the whole LIP that stand-by validators never exist..